helper/include/odp/helper/odph_debug.h

```diff
@@ -63,25 +63,11 @@ typedef enum odph_log_level {
  */
 #define ODPH_LOG(level, fmt, ...) \
 do { \
-	switch (level) { \
-	case ODPH_LOG_ERR: \
+	if (level != ODPH_LOG_DBG || ODPH_DEBUG_PRINT == 1) \
 		fprintf(stderr, "%s:%d:%s():" fmt, __FILE__, \
```

Comment:
Matter of taste, but there could be a whitespace after the location info -> `"%s:%d:%s(): "`, makes the entries slightly more readable.

Reply:
Agreed, this would make reading log messages easier. The rest of the print is application specific anyway, so this change shouldn't affect any reasonable log parsing.

PR needs rebase to pass CI.